controller/test/sensor_tests/sensors_test.cpp

```diff
@@ -111,3 +115,35 @@ TEST(SensorTests, TestPositiveVolumetricFlowCalculation) {
   EXPECT_NEAR(Sensors::PressureDeltaToFlow(kPa(-100.0f)).cubic_m_per_sec(),
               -9.72e-3f, COMPARISON_TOLERANCE_FLOW);
 }
+
+static constexpr Time base = millisSinceStartup(86400000);
+static constexpr Duration sample_period = milliseconds(10);
```

Comment:
This still 2x larger than the integration period in the code, should there be some testing that it behaves correctly when sampled more frequently?

Reply:
Adding some right now, will update in a separate PR

Reply:
Okay if it's coming in an immediate followup then OK to merge this one. Thanks!

Reply:
Or maybe update in this one, as I guess this is where those tests will end up being useful

Reply:
Was this fixed?

Reply:
Added a test at the end which does oversampling